conductor.go

```diff
@@ -20,15 +21,10 @@ type Conductor struct {
 
 // Play starts all the players and gracefully shuts them down
 func (c *Conductor) Play(ctxMain context.Context) error {
-	if c.playing == nil {
-		c.playing = map[string]struct{}{}
-	}
-
-	var wg sync.WaitGroup
-	var lock sync.RWMutex
+	c.playing = map[string]struct{}{}
 
 	// This will be sent to the sub daemons and canceled when the main context ends
-	ctxWthCancel, cancel := context.WithCancel(context.Background())
+	ctxWthCancel, cancel := context.WithCancel(ctxMain)
```

Comment:
Makes sense that the parent can just cancel the context sent to the Players.

In that case, the other context is redundant, we can simply pass `ctxMain` to the players.